docs/02-for-app-authors/12-linter.md

```diff
@@ -59,27 +59,27 @@ and the first `{tld}` component does not start with `io` for github
 and gitlab; `page` for codeberg.
 
 Please follow the [app id guidelines](/docs/for-app-authors/requirements#application-id)
-and [flatpak conventions](https://docs.flatpak.org/en/latest/conventions.html#application-ids)
+and [Flatpak conventions](https://docs.flatpak.org/en/latest/conventions.html#application-ids)
 for more information.
 
 ### appstream-external-screenshot-url
 
 **Exceptions allowed**: No
 
-The screenshots in [appinfo catalogue](https://www.freedesktop.org/software/appstream/docs/chap-CatalogData.html)
+The screenshots in [AppStream Catalog data](https://www.freedesktop.org/software/appstream/docs/chap-CatalogData.html)
 file which are not of [type](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-screenshots)
 `source` aren't mirrored to https://dl.flathub.org/repo/screenshots.
 
-The catalogue file is automatically generated when [building the flatpak](https://docs.flatpak.org/en/latest/building-introduction.html#flatpak-builder)
+The catalogue file is automatically generated when [building the Flatpak](https://docs.flatpak.org/en/latest/building-introduction.html#flatpak-builder)
 by appstream which also does the mirroring when
 `--mirror-screenshots-url=URL` is passed to
-[flatpak-builder](https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html).
+[Flatpak-builder](https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html).
 
 This can happen when mirroring fails due to spurious connectivity issues
 and is usually resolved by restarting the build.
 
 Externally uploaded apps must ensure `--mirror-screenshots-url=URL`
-was passed to flatpak-builder.
+was passed to Flatpak-builder.
```

Comment:
Wondering if this change is actually correct for the flatpak-builder

Reply:
I think so, when not referring to the command itself.

The page headers on Flatpak docs use "Flatpak Builder".

Also I can go with whatever

Reply:
I guess if you don't mean the command it should be without the dash? Anyway not that important

Reply:
Linkified and removed the dash to make it less ambiguous.